examples/text/document_clustering.py

```diff
@@ -102,7 +102,12 @@
 print(__doc__)
 op.print_help()
 
-(opts, args) = op.parse_args()
+
+def is_interactive():
+    return not hasattr(sys.modules['__main__'], '__file__')
+
```

Comment:
Maybe add a comment somewhere saying that this is an IPython console or Jupyter notebook work-around